### ISSUE TITLE
Print Necessary Metrics at end of Simulation

### ISSUE THREAD

**Issue by slichtenheld** [khaledhassan/eel6591-project#9]
- [ ] calculate individual UE packet loss
- [ ] calculate individual UE avg handover delay
- [ ] calculate overall UE packet loss
- [ ] calculate overall handover delays

**Comment by khaledhassan** [comment]
(real) packet loss = num packets sent - packets received
packets lost as reported by the UDPServer only include sequence numbers that were expected but never received, not any later packets -- in other words, it can only figure this out based on the sequence number of the last packet that was successfully received, not the last sequence number that was sent from the source
